browser/ui/webui/brave_rewards_ui.cc

```diff
@@ -115,6 +116,7 @@ class RewardsDOMHandler : public WebUIMessageHandler,
 
   brave_rewards::RewardsNotificationsService* rewards_notifications_service_;  // NOT OWNED
   brave_rewards::RewardsService* rewards_service_;  // NOT OWNED
+  base::WeakPtrFactory<RewardsDOMHandler> weak_factory_{this};
```

Comment:
Could you initialize this inside the class initializer list instead?

Reply:
sure